main.go

```diff
@@ -116,12 +116,22 @@ has been made so far.
 				panic(errors.WithStack(err))
 			}
 
+			source, err := cmd.Flags().GetBool("source")
```

Comment:
can you check the long description above for `cancel-backfill` and see if it's worth updating as well?

Reply:
doesnt look like it needs to be updated, none of the other flags are actually described in there either, just the logic of what it does.